src/overlay/main.scss

```diff
@@ -17,12 +17,12 @@
         left: 0;
         width: 100%;
         height: 100%;
-        background: $mask-background;
+        background-color: change-color($color: $mask-background, $alpha: $mask-opacity);
```

Comment:
`change-color` isn't defined, is it a css method?

Reply:
Its a SASS method: https://sass-lang.com/documentation/functions/color#change-color

Reply:
do you have any idea about the version of `change-color`? For some reason we have to support node-sass@4.6.0 which depends on libsass - the C++ one - 3.5.0.beta.2

Reply:
```scss
// src/overlay/main.scss line20
background-color: $color-calculate-mask-background;

// src/overlay/scss/variable.scss 
// we don't have this file, you need to create one
$color-calculate-mask-background: change-color($color: $mask-background, $alpha: $mask-opacity);

```